Readability.js

```diff
@@ -1722,6 +1722,23 @@ Readability.prototype = {
   isProbablyReaderable: function(helperIsVisible) {
     var nodes = this._getAllNodesWithTag(this._doc, ["p", "pre"]);
 
+    // Get <div> nodes which have <br> node(s) and append them into the `nodes` variable.
+    // Some articles' DOM structures might look like
+    // <div>
+    //   Sentences<br>
+    //   <br>
+    //   Sentences<br>
+    // </div>
+    var brNodes = this._getAllNodesWithTag(this._doc, ["div > br"]);
+    if (brNodes.length > 1) {
+      var set = new Set();
+      [].forEach.call(brNodes, function(node) {
+        var div = node.parentNode;
+        !set.has(div) && set.add(div);
+      });
+      nodes = [].concat.call(Array.from(set), nodes);
+    }
+
```

Comment:
One more thing,  I've updated the patch[1] when I left the comment[2] few hours ago. I'm afraid maybe you haven't seen it.

Just want to ensure one thing here`var brNodes = this._getAllNodesWithTag(this._doc, ["div > br"]);`[1] is really too expensive for `isProbablyReaderable ` method?

Thanks for the responses. (go to bed first)

[1]: https://github.com/mozilla/readability/pull/310/commits/c9f287ac8ff4bdd782085231151455e34cdbd4d3#diff-06d8d22df421dacde90a2268083424abR1725
[2]: https://github.com/mozilla/readability/pull/310#discussion_r83186465